docs/src/componentDocs/DrawerNavItem/examples/SelectingANavItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Box} from '@mui/material';
+import {SelectingANavItemExample} from './SelectingANavItemExample';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+
+const codeSnippet = `<Drawer activeItem="item2">
+    <DrawerBody>
+        <DrawerNavItem
+            itemID={'item1'}
+            title={'Account'}
+            icon={{family: 'material-community', name: 'account', direction: 'auto'}}
+        />
+        <DrawerNavItem
+            itemID={'item2'}
+            title={'Notification'}
+            icon={{family: 'material-community', name: 'bell', direction: 'auto'}}
+        />
+        <DrawerNavItem
+            itemID={'item4'}
```

Comment:
Instead of `item4` you can give itemID `Localization`. Same for other itemIDs you can update.

Reply:
This is not fixed